build.sbt

```diff
@@ -1,18 +1,15 @@
-lazy val snowplowBigqueryLoader = project.in(file("."))
-  .settings(BuildSettings.commonSettings)
-
 lazy val common = project.in(file("common"))
+  .settings(BuildSettings.commonSettings)
   .settings(Seq(
     name := "snowplow-bigquery-common",
+    crossScalaVersions := Seq("2.12.8", "2.11.12"),
```

Comment:
Any reasons to leave 2.11? I think it was necessary for some time because not all components were 2.12-ready.

Reply:
No idea, I think this was in the other commits aside from bad row commit. Since I cleaned the commit history, they are seen as diff right now. Should I remove the 2.11 from respective commit ?